grpc/channelz/v1/channelz.proto

```diff
@@ -435,9 +436,14 @@ service Channelz {
 }
 
 message GetTopChannelsRequest {
-  // start_channel_id indicates that only channels at or above this id should be
+  // start_channel_id indicates that only channels above this id should be
   // included in the results.
   int64 start_channel_id = 1;
+
+  // If non-zero, the server will return a page of results whose
+  // cardinality is equal to or less than this value. If zero, the
+  // server will choose a reasonable page size.
+  uint32 limit = 2;
```

Comment:
Can we use int64?   We have used int64 in most other parts of this API.  The result size will likely never be that big.